### PR TITLE
Adds note for Chrome OS/CloudReady users

### PR DESCRIPTION
**Pull request by Nathan-Galipeau**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

Added note for Chrome OS/CloudReady users to install git by following the directions from the Digital Ocean Tutorial.

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#22207

### CODE REVIEW

File: foundations/installations/prerequisites.md [hunk 1]
```diff
@@ -200,6 +200,8 @@ Once you have successfully met both of these requirements, you should be able to
 
 **Note for CloudReady users**
 
+Chrome OS/CloudReady users will need to install install Git from source with instructions at this [Digital Ocean tutorial](https://www.digitalocean.com/community/tutorials/how-to-install-git-on-debian-10#installing-git-from-source).
```
Comment (xandora) [inline foundations/installations/prerequisites.md:203]:
```suggestion
Chrome OS/CloudReady users will need to install Git from source by following the instructions at this [Digital Ocean tutorial](https://www.digitalocean.com/community/tutorials/how-to-install-git-on-debian-10#installing-git-from-source).
```

**Review by xandora — CHANGES_REQUESTED** [review]
You seem to have forgotten to fill out the required details in the PR template. Please include those, and a link to the issue this is related to. I have also suggested a couple of changes. Once these are updated, this should be good to accept.

**Comment by xandora** [comment]
The changes look good, but you haven't filled out the PR template. Once that's done I'm happy for this to be merged.

**Comment by Nathan-Galipeau** [comment]
Okay, I apologize for ignoring that on the last commit, it should appear as filled out now.

**Comment by xandora** [comment]
Perfect!

**Pull request merged**